.version.sh

```diff
@@ -12,6 +12,7 @@ echo Version: ${VERSION}
 JSRUNTIME="$(jq ".version = \"${VERSION}\"" js-runtime/package.json)" && echo "${JSRUNTIME}" > js-runtime/package.json
 COMPILER="$(jq ".version = \"${VERSION}\"" compiler/package.json)" && echo "${COMPILER}" > compiler/package.json
 AVM="$(sed "s/^version = .*$/version = \"${VERSION}\"/" avm/Cargo.toml)" && echo "${AVM}" > avm/Cargo.toml
+ANYCLOUD="$(sed "s/^version = .*$/version = \"${VERSION}\"/" anycloud/cli/Cargo.toml)" && echo "${ANYCLOUD}" > anycloud/cli/Cargo.toml
```

Comment:
These two changes are unfortunately not enough, you also need to run `cargo build` similar to below to make sure the `Cargo.lock` file is updated with the new version.

Reply:
got catch